governance/votes/Executive vote - January 24, 2024.md

```diff
@@ -0,0 +1,142 @@
+- Reduce [Proportional Kick Incentive (`chip`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-proportional-kick-incentive) by 0.1% from 0.1% to **0%**.
+- Increase [Liquidation Ratio (`mat`)](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) by 9,850% from 150% to **10,000%**.
+
+### SBE Parameter Changes
+
+ **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-4/23441/2)\
+ **Proposal:** [Forum post "Smart Burn Engine - Transaction Analysis & Parameter Reconfiguration Update #4"](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-4/23441)
+
+ If this executive proposal passes, following the recommendation of the Stability Scope Advisory Council Member, BA Labs, the following SBE parameters will be changed:
+
+- The [`bump` (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) parameter will be increased by 20,000 DAI, from 30,000 DAI to **50,000 DAI**.
+- The [`hop` (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) parameter will be increased by 10,512 seconds , from 15,768 seconds to **26,280 seconds**.
+
+### Approve HV Bank (RWA009-A) DAO Resolution
+
+**Authorization:** Ecosystem Approval [1](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/23), [2](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/16)\
+**Proposal:** Forum posts [1](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/20), [2](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/15)
+
+If this executive proposal passes, a DAO resolution pertaining to the HV Bank vault will be approved to instruct the Director of RWAF to take all required actions to facilitate the return of $50,395,016.48 or other amount deemed to be excess cash under the MPA back to Maker, via Galaxy as the exchange agent. The Director of RWAF will be instructed to direct Galaxy to, upon receipt of the Return Amount, convert it into USDC and transfer the USDC to the Urn Input Conduit deployed at [0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1](https://etherscan.io/address/0x08012ec53a7fabf6f33318dfb93c1289886ebbe1).
+
+To execute this process, the following action will be taken if the executive proposal passes:
+
+- The DAO Resolution with IPFS hash [QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w](https://ipfs.io/ipfs/QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w) will be approved.
+- `RWA009_A_INPUT_CONDUIT_URN_USDC` deployed at [0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1](https://etherscan.io/address/0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1) will be added to the chainlog on Mainnet.
```

Comment:
@0xldr, I'm also incorporating this description as part of the final addition to the Exec Sheet.

```suggestion
- `RWA009_A_INPUT_CONDUIT_URN_USDC` deployed at [0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1](https://etherscan.io/address/0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1) will be added to the chainlog on Mainnet.

Further, <conduit>.rely(MCD_ESM) will be called on the SwapInputConduit contracts to allow the ESM module to deny the pause proxy. In the event of a successful governance attack, this prevents funds from being stolen from the SwapInputConduit contracts.
```